--- patch/Makefile.original	2026-05-11 20:11:44
+++ dist/Makefile	2026-05-11 20:16:04
@@ -24,10 +24,10 @@
 
 VERSION=2.3.2
 TREE_DEST=tree
-DESTDIR=${PREFIX}/bin
+#DESTDIR=${PREFIX}/bin
 MAN=tree.1
 # Probably needs to be ${PREFIX}/share/man for most systems now
-MANDIR=${PREFIX}/man
+#MANDIR=${PREFIX}/man
 OBJS=color.o file.o filter.o hash.o html.o info.o json.o list.o tree.o unix.o \
      util.o xml.o strverscmp.o
 
@@ -35,12 +35,12 @@
 
 # Linux defaults:
 #LDFLAGS?=-s
-CFLAGS?=-ggdb
-CFLAGS?=-O3
-CFLAGS+=-std=c11 -Wpedantic -Wall -Wextra -Wstrict-prototypes -Wshadow -Wconversion \
-	-Wdiscarded-qualifiers
+#CFLAGS?=-ggdb
+#CFLAGS?=-O3
+#CFLAGS+=-std=c11 -Wpedantic -Wall -Wextra -Wstrict-prototypes -Wshadow -Wconversion \
+#	-Wdiscarded-qualifiers
 # _LARGEFILE64_SOURCE may be considered obsolete
-CPPFLAGS+=-DLARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64
+#CPPFLAGS+=-DLARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64
 
 # Uncomment for FreeBSD:
 #CC=cc
@@ -69,18 +69,20 @@
 
 # Uncomment for MacOS:
 # It is not allowed to install to /usr/bin on MacOS any longer (SIP):
-#CC = cc
-#CFLAGS?=-O2
-#CFLAGS+=-Wall -fomit-frame-pointer -no-cpp-precomp
-#LDFLAGS+=
+CC = cc
+CFLAGS?=-O2
+CFLAGS+=-Wall -fomit-frame-pointer -no-cpp-precomp
+LDFLAGS+=
 #MANDIR=${PREFIX}/share/man
+BINDIR=$(DESTDIR)/$(PREFIX)/bin
+MANDIR=$(DESTDIR)/$(PREFIX)/share/mem
 
 # Uncomment for HP/UX:
 #prefix=/opt
 #CC=cc
 # manpage of mbsrtowcs() requires C99 and the two defines
 #CFLAGS+=+w -AC99
-#CPPFLAGS+=-D_XOPEN_SOURCE=500 -D_POSIX_C_SOURCE=200112 
+#CPPFLAGS+=-D_XOPEN_SOURCE=500 -D_POSIX_C_SOURCE=200112
 #LDFLAGS+=
 #MANDIR=${PREFIX}/share/man
 
@@ -122,9 +124,12 @@
 	rm -f $(TREE_DEST) *.o *~
 
 install: tree
-	$(INSTALL) -d $(DESTDIR)
+	#$(INSTALL) -d $(DESTDIR)
+	$(INSTALL) -d $(BINDIR)
 	$(INSTALL) -d $(MANDIR)/man1
-	$(INSTALL) $(TREE_DEST) $(DESTDIR)/$(TREE_DEST); \
+	#$(INSTALL) $(TREE_DEST) $(DESTDIR)/$(TREE_DEST); \
+	#$(INSTALL) -m 644 doc/$(MAN) $(MANDIR)/man1/$(MAN)
+	$(INSTALL) $(TREE_DEST) $(BINDIR)/$(TREE_DEST); \
 	$(INSTALL) -m 644 doc/$(MAN) $(MANDIR)/man1/$(MAN)
 
 distclean:
